about.html

```diff
@@ -0,0 +1,187 @@
+                        <li class="nav-item">
+                            <a class="nav-link color font" href="#">News</a>
+                        </li>
+                        <li class="nav-item" id="nav-btn">
+                            <a class="nav-link active" href="#">Apply Now</a>
+                        </li>
+                    
+                    </ul>
+                
+            </div>
+        </nav> 
+
+    </section>
+
+    <section class="first-content d-flex flex-column align-items-center">
+        <div class="content-img">
+        <img src="./images/first-content.png" alt="about image">
+        </div>
+        
+        <div class="detail-desc mt-5 ">
+            <p class="font m-4">
+            The WLIT Global Hackathon, held every year, is a place where
+                students from all over the world and experts, scholars and 
+                activists from all walks of life who support and practice <br> open culture gather together.
+            The Global Summit brings together the community of experts,academics and activist who comprise the Creative
+        Commons affialiate network in a different country every two years.<br>
+        This year,we hope to expand our invitation list including organization and individuals<span> who want to work
+          with us on shared projects that advance the cause of the Commons,free culture and open knowledge.<br>
+          So if you're active and engaged in the world of open content konwledge - free software advocates ,Wikipedian ,
+          Open Knownledge, galleries,libraries,museums,archives,goverments and foundations, lawyers , and avtivist - we
+          hope you'll consider joining us this year to build a stronger, more vibrant commons together</span> 
+        </p> 
+       </div>
+
+       <div class="contact">
+        <p class="p-contact mt-5 font text-center w-4">Please contact us per Email for any further questions about WLIT Global Hackathon 2021
+          !<br><strong>sdsummit2021@sdkosova.org</strong></p>
+      </div>
+    </section>
+
+      <section class="about-second-section d-flex flex-column align-items-center m-5">
+        <h3 class="font color">Software Developers Global Hackathons 2021 Logo</h3>
+        <p>The logo of SD Global Summit 2021 was decided throught the logo competition form 8.June to 7.July.
+        </p>
+        <img src="./images/wlit_logo.png" alt="logo-image" class="wlit-logo">
+        <hr>
+      </section>
+    
+      <section class="events d-flex flex-column align-items-center m-5">
+        <h4 class="font">See the past Global Hackathons</h4>
+        <p>Take a look at the last two SD Global Summits which took place in Buenos Aires and in Warsaw.</p>
+        <div class="events-images d-flex m-5">
+          <img src="./images/event2.png" alt="first-event" class="first-event">
+          <img src="./images/event1.png" alt="second-event" class="second-event">
+        </div>
+      </section>
```

Comment:
- [ ] This section's design for the mobile version of the about page is not in line with the provide design. Look at the differences in the screenshot below and make necessary adjustments.

| The Provide design | Your Implementation |
| --- | --- |
| ![past-gloabal-design](https://user-images.githubusercontent.com/45002654/133676492-2b146b9c-d83d-4ef1-bebb-554bdb160bee.PNG) | ![past-global-imp](https://user-images.githubusercontent.com/45002654/133676585-6583e327-10f9-4db8-8638-8c5e06bc7f37.PNG) |

- Text alignment for sections of the about page should be center in the mobile version.
- The two banners are not properly sized and position as in the provided design.